website/docs/r/edgegateway.html.markdown

```diff
@@ -60,5 +63,17 @@ The following arguments are supported:
 * `advanced` - (Optional) True if the gateway uses advanced networking. Default is `true`.
 * `ha_enabled` - (Optional) Enable high availability on this edge gateway. Default is `false`.
 * `distributed_routing` - (Optional) If advanced networking enabled, also enable distributed routing. Default is `false`.
+* `lb_enabled` - (Optional) Enable load balancing. Default is `false`.
```

Comment:
Currently these new optional LB parameters in resource definition don't have default values, but we are saying three of them are false by default. I suspect it becomes false if undefined? What about actually adding a default = false, so the behavior has less assumptions? Or would that introduce something unwanted?

Reply:
I experimented with that. One thing I didn't like:
If I set `default` I can't make the check if value `is set` in config or is it a `default`. Then it always tries to configure load balancer (with default values if not set).
The actual default configuration at least on 9.5 (but I think others as well) after creation is enabled=false, lb_acceleration_enabled=false, lb_logging_enabled=false, lb_loglevel='info'.
I could set such defaults, then after initial read it wouldn't complain as these values coincide with defaults, but if in some future version of vCD the defaults change then I would start overriding them.

Reply:
Changed these fields to always have default values (same as edge gateway currently gets during provisioning):
enabled=false
lb_acceleration_enabled=false
lb_logging_enabled=false
lb_loglevel='info'

The only scenario when they are ignored is when edge gateway is _not_ advanced.
It also prints a warning if someone is trying to enable it on `simple` edge gateway during `updates procedures and the website docs have a note now.